room_assistance/indico_room_assistance/migrations/20190220_1138_b160812c186b_add_startup_assistance_requests_table.py

```diff
@@ -0,0 +1,90 @@
+"""Add room assistance requests table
+
+Revision ID: b160812c186b
+Revises:
+Create Date: 2019-02-20 11:38:08.691528
+"""
+
+from __future__ import unicode_literals
+
+from alembic import context, op
+from terminaltables import AsciiTable
+
+
+# revision identifiers, used by Alembic.
+revision = 'b160812c186b'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    if context.is_offline_mode():
+        raise Exception('This upgrade is only possible in online mode')
+
+    conn = op.get_bind()
+    conn.execute('''
+        INSERT INTO events.requests (event_id, type, state, data, created_by_id, created_dt)
+        SELECT
+            rvl.linked_event_id,
+            'room-assistance',
+            1,
+            '{}',
```

Comment:
This goes kaboom when trying to view the request details page in the given event. We shouldn't migrate this with empty data...